diff --git a/Cargo.lock b/Cargo.lock
index 6d5ecd255..34a8bc3e5 100644
--- a/Cargo.lock
+++ b/Cargo.lock
@@ -2414,8 +2414,3 @@ dependencies = [
  "cc",
  "libc",
 ]
-
-[[patch.unused]]
-name = "cmake"
-version = "0.1.48"
-source = "git+https://github.com/messense/cmake-rs.git?branch=cross-compile#15ab68baca7c44bcf23c5d91a4201f10d40552a1"
diff --git a/Cargo.toml b/Cargo.toml
index a04d0fcf3..90af0824b 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -159,16 +159,12 @@ features = [
   "date_offset",
 ]
 
-[patch.crates-io]
-cmake = { git = "https://github.com/messense/cmake-rs.git", branch = "cross-compile" }
-
 [lib]
 name = "polars"
 crate-type = ["cdylib"]
 
 [profile.release]
-codegen-units = 1
-lto = "fat"
+lto = "thin"
 
 # This is ignored here; would be set in .cargo/config.toml.
 # Should not be used when packaging
